packages/lib/src/confirm/confirm.wc.svelte

```diff
@@ -1,57 +1,57 @@
-<script>
-    import { clickOutside } from '../clickOutside';
-
-    export let open = false;
-</script>
-
-{#if open}
-    <div class="overlay">
-        <div class="dialog" use:clickOutside on:click_outside={() => open = false}>
-            <div class="flex justify-between items-center gap-4 border-b p-4">
-                <h1>Confirm</h1>
+<svelte:options
+  customElement={{
+    tag: 'jp-confirm',
+    shadow: 'none'
+  }}
+/>
 
-                <button on:click={() => open = false} class="w-6 h-6">
-                <span class="material-symbols-outlined">
-                close
-                </span>
-                </button>
-            </div>
-
-            <p class="p-4" style="max-width: 32ch;">
-                Lorem ipsum dolor sit amet, consectetur adipisicing elit.
-            </p>
+<script>
+  import { clickOutside } from '../clickOutside';
+  import { createEventDispatcher } from 'svelte';
 
-            <div class="flex border-t divide-x">
-                <button class="flex-1 p-4 font-bold text-blue-500" on:click={() => open = false}>Cancel</button>
-                <button class="flex-1 p-4 font-bold text-red-500">Delete it</button>
-            </div>
-        </div>
-    </div>
-{/if}
+  export let title = '';
+  export let message = '';
 
-<style>
-    .overlay {
-        z-index: 10;
-        position: fixed;
-        top: 0;
-        left: 0;
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        width: 100vw;
-        height: 100vh;
-        background-color: rgba(0,0,0,.2);
-    }
+  export let reject = '';
+  export let accept = '';
 
-    .dialog {
-        min-width: 300px;
-        background-color: var(--fg);
-        box-shadow: 0 3px 6px rgba(0,0,0,.16);
-        border-radius: .5rem;
-    }
+  const dispatch = createEventDispatcher();
+</script>
 
-    h1 {
-        font-size: 1.5rem;
-        font-weight: bold;
-    }
-</style>
\ No newline at end of file
+<svelte:window on:keydown|preventDefault={(e) => {
+    if(e.key == 'Escape') dispatch('close')
+}} />
```

Comment:
Don't we need to check if `closable` is `true`?

Reply:
If `closable` is `false`, the event listener on `'close'` does not get attached.

Is it a good practice or should I change the approach?

Reply:
That's great and it should stay but the problem is the component should be usable without the utility function, If someone want's to integrate it with their system directly.